src/Modules/task.js

```diff
@@ -0,0 +1,16 @@
+class Tasks {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+const tasksorting = () => {
+  let tasklist = [];
+  const tasklist1 = new Tasks('wash the dishes', true, 2);
+  const tasklist2 = new Tasks('complete the To Do List Project', false, 1);
```

Comment:
- [ ] [OPTIONAL] Kindly ensure that you follow the correct [naming](https://www.syncfusion.com/blogs/post/10-javascript-naming-conventions-every-developer-should-know.aspx) guidelines of Javascript. This will make your code easily understandable and easy to maintain ☺️ .